src/mscorlib/corefx/Interop/Unix/Interop.Libraries.cs

```diff
@@ -4,15 +4,4 @@
 
 internal static partial class Interop
 {
-    private static partial class Libraries
-    {
-        internal const string GlobalizationInterop = "System.Globalization.Native"; // CoreFX wrappers for ICU
-        // Shims
-        internal const string SystemNative = "System.Native";
```

Comment:
Yes, I just hadn't got to that.

---
In reply to: [99095195](https://github.com/dotnet/coreclr/pull/9269#discussion_r99095195) [](ancestors = 99095195)